src/v0/sources/shopify/transform.js

```diff
@@ -208,6 +208,7 @@ const isIdentifierEvent = (event) =>
   ['rudderIdentifier', 'rudderSessionIdentifier'].includes(event?.event);
 const processIdentifierEvent = async (event, metricMetadata) => {
   if (useRedisDatabase) {
+    const cartToken = event.cartToken.split('?')[0];
```

Comment:
sanitise the input as operation is being performed on string now

Reply:
done